lib/src/util/dart_type_utilities.dart

```diff
@@ -176,12 +176,12 @@ class DartTypeUtilities {
 
   static bool isClass(DartType type, String className, String library) =>
       type != null &&
```

Comment:
`is InterfaceType`, and then unprotected accesses.

Reply:
👍